src/services/iasql.ts

```diff
@@ -796,12 +796,12 @@ export async function install(moduleList: string[], dbId: string, dbUser: string
   // Check to make sure that all dependent modules are in the list
   let missingDeps: string[] = [];
   do {
-    missingDeps = mods
+    missingDeps = [...new Set(mods
```

Comment:
This is needed because now ec2 installs elb as a dependency if not defined, and both share vpc as an inner dependency and it creates a duplicate in the mods array.